jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer-prototype.c

```diff
@@ -92,23 +92,23 @@ ecma_builtin_arraybuffer_prototype_object_slice (ecma_value_t this_arg, /**< thi
 {
   if (!ecma_is_value_object (this_arg))
   {
-    return ecma_raise_type_error (ECMA_ERR_MSG ("Argument 'this' is not object."));
+    return ecma_raise_type_error (ECMA_ERR_MSG ("Argument 'this' is not object"));
```

Comment:
This varies in the code base, and would be good to decide. Put a/an or put nothing. I like the latter, but engines like v8 prefer the former, so the best choice is debatable. What should we do?

Reply:
I prefer "an object", because that's the grammatically correct phrase. These messages are to inform humans, so they should (try to) follow language rules.